tests/apm/mod.rs

```diff
@@ -57,7 +57,8 @@ fn logging() {
     let _ = fs::remove_file("test_apm_log.txt");
 
     // Reset State
-    let reset_client = Client::connect("localhost", 27017).expect("Failed to connect to localhost:27017");
+    let reset_client = Client::connect("localhost", 27017)
```

Comment:
The most recent large rustfmt/clippy PR only checked `src` (and not `tests`), so rustfmt picked up a few things.